### PR TITLE
ClassLoader in fetching service

### PR DESCRIPTION
**Pull request by sudo0rw3ll**
Hello team,
In this task, I've added class loader to load resources needed for bootstrap data and fetching services.
Changes are similar to ones made in market service. I only had to add DockerFile into fetching service
folder structure to make it possible to build and push this image to my docker hub.

Also, docker-compose for prod is changed, it includes fetching service now and appropriate configuration
is made for it. 

deploy.yml file is changed to contain additional steps of building and publishing fetching service image to docker hub.

Please, go through all files and ping me if any changes need to be made.

Best,
Vid

### CODE REVIEW

File: fetching-service/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+# - "-f $HOME/pom.xml" - putanja do pom.xml fajla
+RUN --mount=type=cache,target=/root/.m2 mvn -DfinalName=app -Dmaven.test.skip -Dcheckstyle.skip=true -f $HOME/pom.xml clean package
+
+# Ovo je drugi stage gde dobijamo rezultirajuci image.
+# Ovde kopiramo .jar fajl napravljen u prethodnom, tj. prvom stage-u.
+FROM openjdk:23
+
+COPY --from=builder /usr/app/target/app.jar /app.jar
+
+EXPOSE 8081
```
Comment (MarkoNik) [inline fetching-service/Dockerfile:38]: zar ne treba 8082?

**Review by Kovelja009 — APPROVED** [review]

File: fetching-service/src/main/java/rs/edu/raf/banka1/utils/Constants.java [hunk 2]
```diff
@@ -4,26 +4,28 @@
...
-    public static final String listingsFilePath = getAbsoluteFilePath(
-            "listings.json");
-    public static final String businessHoursFilePath = getAbsoluteFilePath(
-            "working_hours_and_holidays_for_exchanges.json");
-    public static final String micCsvFilePath = getAbsoluteFilePath(
-            "ISO10383_MIC.csv");
-    public static final String countryTimezoneOffsetsFilePath = getAbsoluteFilePath(
-            "country_timezone_offsets.json");
+    public static final String listingsFilePath =
+            "listings.json";
```
Comment (MarkoNik) [inline fetching-service/src/main/java/rs/edu/raf/banka1/utils/Constants.java:14]: cini mi se da nisi izmenio deo gde se ucitava ovaj fajl

**Review by MarkoNik — CHANGES_REQUESTED** [review]

File: fetching-service/src/main/java/rs/edu/raf/banka1/utils/Constants.java (hunk 2 above)
Comment (MarkoNik) [reply fetching-service/src/main/java/rs/edu/raf/banka1/utils/Constants.java:14]: listings.json

File: fetching-service/Dockerfile (hunk 1 above)
Comment (sudo0rw3ll) [reply fetching-service/Dockerfile:38]: Ispravljam

File: fetching-service/src/main/java/rs/edu/raf/banka1/utils/Constants.java (hunk 2 above)
Comment (sudo0rw3ll) [reply fetching-service/src/main/java/rs/edu/raf/banka1/utils/Constants.java:14]: Dodato, baci proveru

Comment (MarkoNik) [reply fetching-service/src/main/java/rs/edu/raf/banka1/utils/Constants.java:14]: baci pogled ako se popunjava tabela sa stockovima onda smo dobri

**Pull request merged**